code/index.html

```diff
@@ -2,17 +2,131 @@
 <html>
   <head>
     <meta charset="utf-8" />
-    <title>Project Name</title>
+    <title>The Cat Working Space</title>
     <link rel="stylesheet" href="./style.css" />
-    <!-- dont forget the meta tag for responsiveness -->
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    
   </head>
   <body>
+
     <header>
-      <!-- If you want a header with a title, logo and a navbar, here is where you should add that. -->
+
+      <div class="hamburger">
+        <div></div>
+        <div></div>
+        <div></div>
+      </div>
+      <img src="/assets/logo.png" alt= "Katze">
+
+      <nav>
+        <a href="#"> Reservation</a>
+        <a href="#"> Über</a>
+        <a href="#"> Kontakt</a>
+      </nav>
+    
     </header>
 
+    <section class="first-container">
+      <section class="container"
+        <img src="/assets/header.JPG" alt="katze bild">
+          <div> 
+    
+            <h1> The Cat Working Space </h1>
+            <h2> Unter nicen Kollegen</h2>
+          </div>
+        </section>
+      </section>
+
+
+        <section class="box-container">
+
+          <section class="container">
```

Comment:
I'm not sure you need both of these containers as they are both wrapping the same amount of elements.